spec/features/orders/show_spec.rb

```diff
@@ -6,11 +6,12 @@
 
     @merchant_1 = create(:merchant)
 
-    @item_1 = @merchant_1.items.create!(attributes_for(:item))
-    @item_2 = @merchant_1.items.create!(attributes_for(:item))
+    @item_1 = @merchant_1.items.create!(attributes_for(:item, inventory: 10))
+    @item_2 = @merchant_1.items.create!(attributes_for(:item, inventory: 10))
```

Comment:
Did you have to pass in this inventory value to avoid certain scenarios? Doesn't factory bot generate a random integer (between 1 and 100)?